python/hail/keytable.py

```diff
@@ -1076,3 +1077,51 @@ def import_fam(path, quantitative=False, delimiter='\\\\s+', missing='NA'):
         hc = Env.hc()
         jkt = Env.hail().keytable.KeyTable.importFam(hc._jhc, path, quantitative, delimiter, missing)
         return KeyTable(hc, jkt)
+
+    def union(self, *kts):
```

Comment:
I did, and I was right that you don't have the class before the definition. I can add lazy typecheck evals but want to do that in a separate branch